libraries/botbuilder/src/skills/skillHandler.ts

```diff
@@ -218,7 +221,11 @@ export class SkillHandler extends ChannelServiceHandler {
         AppCredentials.trustServiceUrl(skillConversationReference.conversationReference.serviceUrl);
 
         await (this.adapter as BotFrameworkAdapter).continueConversation(skillConversationReference.conversationReference, skillConversationReference.oAuthScope, callback);
-        return { id: uuid() };
+        
+        if (!resourceResponse) {
+            resourceResponse = { id: uuid() };
```

Comment:
Can you check with @johnataylor and make sure the implementation is consistent with what he is implementing in dotnet?

Reply:
Checked the dotnet version and this looks good.